frontends/web/src/routes/bitboxbase/bitboxbase.tsx

```diff
@@ -225,9 +247,9 @@ class BitBoxBase extends Component<Props, State> {
         this.setState({ password });
     }
 
-    private submitChangePassword = (event: Event) => {
+    private submitChangePasswordSetup = (event: Event) => {
         event.preventDefault();
-        apiPost(this.apiPrefix() + '/user-change-password', {username: 'admin', newPassword: this.state.password})
+        apiPost(this.apiPrefix() + '/user-change-password', {username: 'admin', password: 'ICanHasPasword?', newPassword: this.state.password})
```

Comment:
is `ICanHasPasword?` the default pw? is so please put it into a descriptive variable, it looks like forgotten debug code otherwise :D